packages/video_player/video_player/lib/video_player.dart

```diff
@@ -145,8 +151,9 @@ class VideoPlayerValue {
         'buffered: [${buffered.join(', ')}], '
         'isPlaying: $isPlaying, '
         'isLooping: $isLooping, '
-        'isBuffering: $isBuffering'
+        'isBuffering: $isBuffering, '
```

Comment:
This is a good change, it shouldn't be breaking (I hope nobody is parsing the toString of this controller)